### PR TITLE
Implemented optional Dark Bremsstrahlung process into custom physics

### PR DESCRIPTION
**Pull request by revering**
Implemented dark bremsstrahlung physics process into CustomPhysics.
Backport of dark bremsstrahlung implementation from pull #38329 to CMSSW_12_4_X for run 3 MC generation.

### CODE REVIEW

**Comment by qliphy** [comment]
please test

**Comment by perrotta** [comment]
verbatim backport of https://github.com/cms-sw/cmssw/pull/38329

File: SimG4Core/CustomPhysics/interface/APrimePhysics.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#ifndef SIMAPPLICATION_APRIMEPHYSICS_H_
+#define SIMAPPLICATION_APRIMEPHYSICS_H 1_
```
Comment (civanch) [inline SimG4Core/CustomPhysics/interface/APrimePhysics.h:2]: @revering , there was a fix in #38329 on top of your PR, please use it.

**Comment by perrotta** [comment]
backport of https://github.com/cms-sw/cmssw/pull/38329

File: SimG4Core/CustomPhysics/interface/APrimePhysics.h (hunk 1 above)
Comment (forthommel) [inline SimG4Core/CustomPhysics/interface/APrimePhysics.h:2]: Same issue as reported in original PR and fixed in #38862.

**Comment by makortel** [comment]
Please include https://github.com/cms-sw/cmssw/pull/38874 as well in the backport.

**Comment by revering** [comment]
@makortel @forthommel Fixes now included, thank you for the careful review!

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**